python3.12/apigw-pytorch/{{cookiecutter.project_name}}/app/requirements.txt

```diff
@@ -1,4 +1,4 @@
--f https://download.pytorch.org/whl/torch_stable.html
-torch=={{cookiecutter.pytorch_version}}+cpu
-torchvision=={{cookiecutter.torchvision_version}}+cpu
+-f https://download.pytorch.org/whl/cpu
```

Comment:
I guess maybe this just works (the tests pass, right?), but the example from StackOverflow (https://stackoverflow.com/a/78215213/574808) has
```
--index-url https://download.pytorch.org/whl/cpu
--extra-index-url https://pypi.org/simple
```
instead of
```
-f https://download.pytorch.org/whl/cpu
```

The intention that the *first* index is the `cpu` index. And you add "the regular" index as a secondary. The point is that when you just ask for a version of Torch, you will get the `cpu` version of it because it will be on your first index. 

I think that if we just add it as a `-f` ([--find-links](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-f)), then it might not get the `cpu` version by default. I would prefer to do the version from StackOverflow.